numba/core/ir_utils.py

```diff
@@ -764,9 +790,11 @@ def is_pure(rhs, lives, call_table):
             if func_name not in call_table or call_table[func_name] == []:
                 return False
             call_list = call_table[func_name]
-            if (call_list == [slice] or
-                call_list == ['log', numpy] or
-                call_list == ['empty', numpy]):
+            if (
+                call_list == [slice]
+                or call_list == ['log', numpy]
+                or call_list == ['empty', numpy]
+            ):
```

Comment:
Does this help with `flake8`?

Reply:
See [above comment](https://github.com/numba/numba/pull/7195#discussion_r706624366).